app/views/conversations/show.html.erb

```diff
@@ -37,7 +37,15 @@
   <div class="request-container">
     <div class="conversation-request-card">
       <div class="conversation-request-card-status">
-        <p id="request-status"><%= @request.status.capitalize %></p>
+        <% if @request.status != 0 %>
```

Comment:
Ok, one last thing. I was just testing this and it seems that if we write it that way it never changes when I change the status. If I change the code to read `<% if @request.status != "pending" %>` then it works. Do you see the same behavior?

Reply:
Correct, thanks: I didn't notice.
Just pushed this change